data-catalog/base/hue/hue-ini-secret.yaml

```diff
@@ -109,6 +109,7 @@ stringData:
       auth_username=hue
       auth_password=password
       hive_conf_dir=/etc/hive/conf
+      server_conn_timeout=200
```

Comment:
I haven't looked at the plaintext version, but I think it may be necessary to set this in the overlay

Reply:
oh, you're right. Sorry about that.

I'm sending a change soon.

Reply:
you will also have to change this in the stage and prod overlays, I cant see it in the prod one
![image](https://user-images.githubusercontent.com/22432723/102395156-c493ac80-3fa8-11eb-9d7e-d05dc14f63b1.png)